core-bundle/src/Resources/contao/modules/ModuleQuicknav.php

```diff
@@ -130,6 +130,7 @@ protected function getQuicknavPages($pid, $level=1, $host=null)
 				$objSubpage->domain = $host;
 			}
 
+			$objSubpage->loadDetails();
```

Comment:
`->domain` gets overwritten by `loadDetails()` so we have to move this before line 127 IMO.

Reply:
Good catch. 👍🏻 Fixed in 21f340966cd0f2c06d649b8f6ce7b94e2eee3faa.